libs/brig-types/src/Brig/Types/User.hs

```diff
@@ -246,6 +252,7 @@ data NewUser = NewUser
     , newUserLocale         :: !(Maybe Locale)
     , newUserPassword       :: !(Maybe PlainTextPassword)
     , newUserExpiresIn      :: !(Maybe ExpiresIn)
+    , newUserManagedBy      :: !(Maybe ManagedBy)
```

Comment:
Just curious; do we tend to make all data strict? I presume it makes sense in most (if not all) cases; has anyone considered using a global `StrictData` pragma?

Reply:
> do we tend to make all data strict

Yes.

> has anyone considered using a global `StrictData` pragma?

I recall considering it when I first got here. Then I got scared of enabling it and decided not to spend more time on figuring out whether it's safe or not. I don't have any specific concerns in mind, though.

Reply:
@neongreen It would either speed things up; slow things down, or neither haha; tough to say which without trying it.

Reply:
I'm just slightly worried that we might be relying on laziness in some place or other.

Reply:
It's certainly possible; but if that's true the result will be either that the integration tests fail; or that we discover some place where we use laziness unexpectedly; If we do require it in some place we can explicitly denote it with `~` to make it lazy again;

Granted, this maybe just takes time that nobody has; just curious :)